.rubocop.yml

```diff
@@ -47,3 +47,15 @@ Metrics/LineLength:
     - app/controllers/application_controller.rb
     - app/controllers/listings_controller.rb
     - db/migrate/**
+
+Metrics/CyclomaticComplexity:
+  Exclude:
+    - app/controllers/listings_controller.rb
+
+Metrics/PerceivedComplexity:
+  Exclude:
+    - app/controllers/listings_controller.rb
+
+Style/GuardClause:
+  Exclude:
+    - app/controllers/listings_controller.rb
```

Comment:
This was making the code harder to understand.

Reply:
@dladowitz We can use the `MinBodyLength` setting to make this rule a little more friendly with smaller `if` blocks:

```rb
Style/GuardClause: { MinBodyLength: 3 }
```

That way it'll only complain for `if` blocks that are 3 or more lines.